server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -801,14 +811,24 @@ public void run()
               && coordLeaderSelector.isLeader()
               && startingLeaderCounter == coordLeaderSelector.localTerm()) {
 
+            final long start = System.nanoTime();
             params = duty.run(params);
+            final long end = System.nanoTime();
 
             if (params == null) {
               // This duty wanted to cancel the run. No log message, since the duty should have logged a reason.
               return;
+            } else {
+              params.getCoordinatorStats().addToDutyStat("runtime", duty.getDutyAlias(), TimeUnit.NANOSECONDS.toMillis(end - start));
```

Comment:
will it be ok to just use "duty.getClass().getName()" for this use case instead of extending the interface?

Reply:
ya, I realized that would probably have been better after I looked at what I ended up with for alias values